lib/models/Instance.js

```diff
@@ -9,7 +9,7 @@ module.exports = Genesis.create("instance", {
 		id    : Joi.string().required(),
 		type  : Joi.string().optional().default("t2.micro"),
 		ami   : Joi.string().optional().default("ami-d05e75b8"),
-		state : Joi.string().regex(/pending|ready|failed/).optional().default("pending"),
+		state : Joi.string().regex(/\bfailed\b|\bready\b|\bpending\b/).optional().default("pending"),
```

Comment:
Does this imply that the `state` can have multiple values? Is there any reason that we wouldn't use Joi's [`allow()`](https://github.com/hapijs/joi#anyvalidvalue---aliases-only-equal) for this? Perhaps something like `Joi.string().allow("failed", "ready", "pending")`?